src/test/bundle_test.ts

```diff
@@ -391,4 +394,108 @@ suite('BuildBundler', () => {
     // We should not have the inlined file in the output.
     assert.isNotOk(getFile('framework.html'));
   });
+
+  suite('options', () => {
+
+    const projectOptions = {
+      root: 'test-fixtures/test-project',
+      entrypoint: 'index.html',
+      fragments: ['shell.html'],
+      sources: ['index.html', 'shell.html', 'source-dir/my-app.html']
+    };
+
+    test('excludes: a file', async() => {
```

Comment:
I generally prefer that 1) the nested suite names + test name read mostly like a sentence that describes the intention of the test 2) The suite names mostly refer to concrete elements in the source - libraries, functions, classes, methods, etc. 3) lower priority, avoid too much prose to make the combined description hard to parse, use terse, list-item like wording.

In this case the tests are running through each option, so for readability I like having them all start with the option name. It's easier to parse the whole suite that way.